### PR TITLE
Experiment: Install local testing env

### PR DESCRIPTION
**Pull request by Nevoss**
ref: ED-2735

### CODE REVIEW

**Comment by Nevoss** [comment]
@ronkelementor

**Review by matipojo — APPROVED** [review]

**Comment by ronkelementor** [comment]
@Nevoss 
1. Please add README file
2. Does it working on Windows machine?

**Comment by Nevoss** [comment]
@matipojo @ronkelementor Please re-review this PR.

File: docs/devlopment/composer.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Composer
+
+A quote from composer website that describes the tool ([https://getcomposer.org](https://getcomposer.org)):
+
+"Composer is a tool for dependency management in PHP. It allows you to declare the libraries your project depends on and it will manage (install/update) them for you."
+
+## Installation
+
+**Note:** Windows users please use a unix shell like `Windows Subsystem for Linux`([https://docs.microsoft.com/en-us/windows/wsl/install-win10](https://docs.microsoft.com/en-us/windows/wsl/install-win10)) or `git bash`([https://git-scm.com/downloads](https://git-scm.com/downloads))
```
Comment (matipojo) [inline docs/devlopment/composer.md:9]: It's easier to use the installer - in order to add it also to the PATH, etc.
https://getcomposer.org/doc/00-intro.md#using-the-installer or use the manual steps:
https://getcomposer.org/doc/00-intro.md#manual-installation

File: docs/devlopment/phpunit.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Installing `svn` in Linux subsystem: `sudo apt-get install subversion`
+
+#### Installation script
+
+Run `composer run test:install`. it should prompt questions about your database credentials make sure to fill correct credentials.
+
+That's all you can run `composer test` which will run the whole "Elementor" test suite, any argument that allowed to pass to `phpunit` command you can pass to `composer test` command, e.g: `composer test -- --filter Elementor_Test_Bootstrap`. just make sure to pass extra ` -- ` before your argument. 
+ 
+ 
+
```
Comment (matipojo) [inline docs/devlopment/phpunit.md:36]: Remove extra lines.

File: docs/devlopment/composer.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+1. Run the commands as described here: [https://getcomposer.org/download/](https://getcomposer.org/download/) (the commands changed from time to time based on the composer version).
+
+2. Move the bin that was created to `usr/bin` directory to allow use composer globally: `mv ./composer.phar /usr/bin/composer` (maybe require sudo permission).
+
+3. Run: `composer -v` and make sure it returns the expected version, if not restart your shell client.
+
+4. `cd` into Elementor root directory and run `composer install`
+
+
```
Comment (matipojo) [inline docs/devlopment/composer.md:19]: Remove extra line.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: docs/devlopment/composer.md (hunk 1 above)
Comment (Nevoss) [reply docs/devlopment/composer.md]: Done

File: docs/devlopment/phpunit.md (hunk 2 above)
Comment (Nevoss) [reply docs/devlopment/phpunit.md]: Done

File: docs/devlopment/composer.md (hunk 3 above)
Comment (Nevoss) [reply docs/devlopment/composer.md]: Done

**Review by matipojo — CHANGES_REQUESTED** [review]
Please pull 3.3.o in order to pass the CI checks

**Review by matipojo — APPROVED** [review]

**Review by ronkelementor — APPROVED** [review]

**Pull request merged**